src/sinks/loki/config.rs

```diff
@@ -65,22 +66,25 @@ pub struct LokiConfig {
     /// The base URL of the Loki instance.
     ///
     /// Vector will append the value of `path` to this.
+    #[configurable(metadata(docs::examples = "http://localhost:3100"))]
     pub endpoint: UriSerde,
 
     /// The path to use in the URL of the Loki instance.
-    ///
-    /// By default, `"/loki/api/v1/push"` is used.
```

Comment:
Does this need to be specified somewhere, or is it magically generated by the serde default?

Reply:
🪄 ❗

Reply:
It's represented by the default value `serde` is instructed to use, and is put into the machine-generated documentation.